examples/alerts/alerts.md

```diff
@@ -612,17 +612,6 @@ rules:
 ```yaml mdox-exec="cat examples/tmp/thanos-component-absent.yaml"
 name: thanos-component-absent
 rules:
-- alert: ThanosBucketReplicateIsDown
-  annotations:
-    description: ThanosBucketReplicate has disappeared. Prometheus target for the
-      component cannot be discovered.
-    runbook_url: https://github.com/thanos-io/thanos/tree/main/mixin/runbook.md#alert-name-thanosbucketreplicateisdown
-    summary: Thanos component has disappeared.
-  expr: |
-    absent(up{job=~".*thanos-bucket-replicate.*"} == 1)
-  for: 5m
-  labels:
-    severity: critical
```

Comment:
Looks like this alert was removed at https://github.com/thanos-io/thanos/pull/3856. Is this an intended change? @kakkoyun

Reply:
Yes, there is another generator that handles the generation of these alerts. This was redundant.